vuepress/docs/.vuepress/next.js

```diff
@@ -1,69 +1,129 @@
 module.exports = {
     docsSidebar: function(path) {
         return [
-            {
-                title: 'Introduction',
-                path: path,
-            },
             {
                 title: 'Getting Started',
                 children: [
                     {
-                        title: 'Overview',
-                        path: path  + 'getting-started/',
+                        title: 'Introduction',
+                        path: path,
                     },
                     {
-                        title: 'Welcome Wizard',
-                        path: path + 'getting-started/welcome-wizard.md'
+                        title: 'Learn About Entando',
```

Comment:
I don't think Learn About Entando is right here - since this is the actual getting started install guide.